jgiwinski/romcom
# Make random cover button function correctly

### What does this PR do?
- Makes the "random button' display random covers with correct Id's, title names, and values
### What does it fix?
- NA
### Is this a feature or a fix?
- Feature
### Where should the reviewer start?
- See line 9
### How should this be tested?
- Open index.html from console and press "show new random cover"   to check if working properly
